src/constants.ts

```diff
@@ -37,3 +37,9 @@ export const DEBUG = false; // Show ship collision boundary and ship center dot
 export const SAVE_KEY_HIGH_SCORE = 'highscore'; // localstorage of high score.
 export const SAVE_KEY_SOUND_ON = 'musicOn'; // localstorage of high score.
 export const SAVE_KEY_MUSIC_ON = 'soundOn'; // localstorage of high score.
+
+/* Drawing Constants*/
+export const TEXT_SIZE = 40; // Text font height in pixels
+export const TEXT_FADE_TIME = 2.5; // text fade in seconds.
+export const CVS = document.querySelector('canvas')!;
```

Comment:
I personally prefer `"strictNullChecks": false,` that you set to true by strict mode.

Reply:
Agreed. I removed all the exclamations and included this option in .tsconfig.